codalab/lib/bundle_cli.py

```diff
@@ -1022,7 +1022,16 @@ def do_workers_command(self, args):
         client = self.manager.current_client()
         raw_info = client.get_workers_info()
 
-        columns = ['worker_id', 'cpus', 'gpus', 'memory_bytes', 'last_checkin', 'tag']
+        columns = [
+            'worker_id',
+            'cpus',
+            'gpus',
+            'memory_bytes',
+            'last_checkin',
+            'tag',
+            'available_disk_space',
```

Comment:
For consistency, append `_bytes`.  I think `free_disk_bytes` is a bit clearer / shorter.